Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2868,8 +2906,7 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 
 bool WebExtensionContext::decidePolicyForNavigationAction(WKWebView *webView, WKNavigationAction *navigationAction)
 {
-    // FIXME: <https://webkit.org/b/246485> Handle inspector background pages in the assert.
-    ASSERT(webView == m_backgroundWebView);
+    ASSERT(webView == m_backgroundWebView || isInspectorBackgroundPage(webView));
```

Comment:
You need to conditionalize the call to `isInspectorBackgroundPage()` on `ENABLE(INSPECTOR_EXTENSIONS)`.